flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/async/AsyncWaitOperator.java

```diff
@@ -209,41 +176,34 @@ else if (element.isLatencyMarker()) {
 	}
 
 	@Override
-	public void processElement(StreamRecord<IN> element) throws Exception {
-		final StreamRecordQueueEntry<OUT> streamRecordBufferEntry = new StreamRecordQueueEntry<>(element);
+	public void processElement(final StreamRecord<IN> element) throws Exception {
```

Comment:
Nit: this change might not be suggested mixing with the core changes. Unless this line was also touched by the core change, then we could add the `final` in passing. From the consistent view,  in the following `processWatermark(Watermark mark)` we also have no final argument.